frontend/src/components/MapView/LeftPanel/layersPanel/MenuItem/MenuSwitch/index.tsx

```diff
@@ -129,6 +129,7 @@ const MenuSwitch = memo(({ title, layers, extent }: MenuSwitchProps) => {
       elevation={0}
       classes={{ root: classes.root }}
       onChange={handleAccordionExpand}
+      TransitionProps={{ unmountOnExit: true }}
```

Comment:
This options allow to not render content until the accordion is open (not good for seo but for us and performance it is better).